docs/DeveloperGuide.md

```diff
@@ -1115,10 +1115,15 @@ Deleting an existing appointment from a patient
 
 Team size: 5
 
-1\. **Allow names to include symbols and special characters**
+1\. **Enhance Name Validation to Support Special Characters and Improve Formatting**
```

Comment:
Could possible just rephrase this to 'Enhance Name Validation' so that it is not interpreted as 2 separate enhancements

Reply:
Amended